src/main/java/hudson/plugins/git/extensions/impl/CloneOption.java

```diff
@@ -148,7 +149,12 @@ public void decorateCloneCommand(GitSCM scm, Run<?, ?> build, GitClient git, Tas
             // in a single job definition.
             RemoteConfig rc = scm.getRepositories().get(0);
             List<RefSpec> refspecs = rc.getFetchRefSpecs();
-            cmd.refspecs(refspecs);
+            List<RefSpec> expandedRefSpecs = new ArrayList<>();
+            EnvVars env = build.getEnvironment(listener);
+            for (RefSpec ref:refspecs) {
```

Comment:
Please use spaces around `:`, they are used in most places in the code, and they make the code more readable.

Reply:
Thanks for the advice, I have corrected it.